WordPress/Classes/ViewRelated/Site Creation/Design Selection/SiteDesignContentCollectionViewController.swift

```diff
@@ -161,6 +171,7 @@ extension SiteDesignContentCollectionViewController: UITableViewDataSource {
         cell.selectionStyle = UITableViewCell.SelectionStyle.none
         cell.section = isLoading ? nil : sections[indexPath.row]
         cell.isGhostCell = isLoading
+        cell.ghostThumbnailSize = ghostThumbnailSize
```

Comment:
@Gio2018 I realized this is a bug if for some reason the `ghostThumbnailSize` differs than what the actual content size is. 

Here's how it can happen, and why it currently only applies to future work when we add recommended designs:

1. Recommended designs will _probably_ be shown, so we set our ghost thumbnail size to the really big ones
2. We run into a scenario where we actually can't show recommended designs (there was no vertical chosen and no Blog category to fall back on)
3. When the `UITableView` goes to grab a new cell it properly sets the `cell.section`, which sets the right height for the content
4. We then immediately overwrite that height by setting the `ghostThumbnailSize` (our assumption from step 1)

One way to fix is is to just not set the ghost thumbnail size if we're not loading. I'll wait until your first review pass before making the update, unless I get your go-ahead before then. 👍

Reply:
This should be resolved with commit 447efa2bd0e74bb255e750a57c5f71b2858870e5.